projects/1.16/assets/materialis/materialis/lang/zh_cn.json

```
@@ -152,42 +219,133 @@
+  "modifier.materialis.lesser_psionizing_radiation.flavor": "你能感觉到它在发光",
+  "modifier.materialis.lesser_spell_casting": "施法",
+  "modifier.materialis.lesser_spell_casting.description": "获得右击使用术式的能力，但失去使用方块破坏和攻击术式的能力",
+  "modifier.materialis.lesser_spell_casting.flavor": "不需要熔化金属",
+  "modifier.materialis.manaburner": "魔力燃烧室",
+  "modifier.materialis.manaburner.description": "消耗魔力加快挖掘速度",
+  "modifier.materialis.manaburner.flavor": "烧穿魔力",
+  "modifier.materialis.manaburner.mining_speed": "有魔力时的挖掘速度",
+  "modifier.materialis.manaripper": "魔力撕裂者",
+  "modifier.materialis.manaripper.attack_damage": "有魔力时的攻击速度",
+  "modifier.materialis.manaripper.description": "消耗魔力增加攻击伤害",
+  "modifier.materialis.manaripper.flavor": "狂暴鸿儒",
+  "modifier.materialis.manashield": "魔力护盾",
+  "modifier.materialis.manashield.description": "有几率消耗魔力来使你的工具不损失耐久度",
+  "modifier.materialis.manashield.flavor": "植物学！",
+  "modifier.materialis.neptunes_blessing": "海王的祝福",
+  "modifier.materialis.neptunes_blessing.description": "对浸泡在液体中的怪物造成更多伤害，并使你在游泳时的挖掘速度与正常状态下相同",
   "modifier.materialis.neptunes_blessing.flavor": "保持水分",
   "modifier.materialis.nocturnal": "夜曲",
   "modifier.materialis.nocturnal.description": "工具在夜间的采掘速度提升，午夜效果最强",
   "modifier.materialis.nocturnal.flavor": "别睡觉了！",
+  "modifier.materialis.nocturnal.mining_speed": "午夜时的挖掘速度",
   "modifier.materialis.old_timer": "旧钟",
   "modifier.materialis.old_timer.description": "工具的耐久度越高，修复消耗的材料越多",
   "modifier.materialis.old_timer.flavor": "锈桶！",
   "modifier.materialis.otherworldly": "彼岸",
   "modifier.materialis.otherworldly.description": "允许采掘彼岸矿物",
   "modifier.materialis.otherworldly.flavor": "来自另一个已知的位面！",
+  "modifier.materialis.overeating": "暴食",
+  "modifier.materialis.overeating.description": "你工具上的过载史莱姆效果突然变得诱人起来",
+  "modifier.materialis.overeating.flavor": "五秒定律！",
   "modifier.materialis.overweight": "超重",
+  "modifier.materialis.overweight.description": "吃掉一些挖掘速度和攻击伤害来获得更多过载史莱姆容量",
   "modifier.materialis.overweight.flavor": "超大容量！",
+  "modifier.materialis.pipe_wrenching": "拧管道",
+  "modifier.materialis.pipe_wrenching.description": "工具可以设置更好的管道",
+  "modifier.materialis.pipe_wrenching.flavor": "真漂亮啊！",
+  "modifier.materialis.pixiecaller": "精灵召唤者",
+  "modifier.materialis.pixiecaller.description": "当你受到伤害时有几率生成一只小精灵",
+  "modifier.materialis.pixiecaller.flavor": "无精灵无真相",
   "modifier.materialis.polished": "抛光",
+  "modifier.materialis.polished.attack_damage": "抛光攻击伤害",
   "modifier.materialis.polished.description": "耐久度越高，攻击伤害越高",
   "modifier.materialis.polished.flavor": "我能从里面看到我的脸！",
+  "modifier.materialis.power_hungry": "电老虎",
+  "modifier.materialis.power_hungry.description": "每级工具每损失1点耐久值消耗100RF，如果没有能量就看起来无法正常运作",
+  "modifier.materialis.power_hungry.flavor": "自大狂！",
+  "modifier.materialis.processor": "中央处理器",
+  "modifier.materialis.processor.description": "使你的工具在使用术式时处理传感器的输入",
+  "modifier.materialis.processor.flavor": "它是加工出来的吗？",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "modifier.materialis.processor.flavor": "处理过了吗？",
```
应是个双关，我也不知道怎么翻，没玩过Psi
原文：**Is this processed?**

---

我也没玩过，但我觉得这和Psi关系不大，也许是process的双义
双关、谐音翻译起来老烦了（哭）国外还老喜欢搞这种